src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
```

Comment:
@timmywil Agreed, added to the list. In fact it'll be testing against `core-js`, not Babel as that's the default polyfill that Babel advises; technically speaking Babel transforms the source code but doesn't define any polyfills by itself.